client/test/who_settings_test.dart

```diff
@@ -25,15 +23,23 @@ void main() {
     );
   }
 
-  testWidgets('WHO Settings Page is rendering its PageHeader properly',
-      (WidgetTester tester) async {
-    // Increasing the default viewport size to avoid RenderFlex overflow error
-    await binding.setSurfaceSize(Size(800, 800));
-    await tester.pumpWidget(testableWidget(
-        child: SettingsPage(
-      notifications: Notifications(service: WhoService(endpoint: '')),
-    )));
-    await tester.pumpAndSettle();
-    expect(find.text('Settings'), findsWidgets);
-  });
+  testWidgets(
+    'WHO About Page is rendering its PageHeader properly',
+    (WidgetTester tester) async {
+      // Increasing the default viewport size to avoid RenderFlex overflow error
```

Comment:
If this checking that nothing is cut off? If so, then would it not make more sense to test this at the smallest screen size that still works?

Reply:
We can set that up -- however I believe it would belong in another PR. This PR is preserving the test as a simple example test.

Reply:
Sounds good to me. Very good to add tests like this. Especially for UI which can be tricky in many different ways.